routes/article.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express')
-const Article = require('../models/ArticleModel')
-const Keyword = require('../models/KeywordModel')
+const Article = require('../models/ArticleModel.js')
```

Comment:
Why did you add the .js extention here? I thought it will assume .js if none is given which to me looked better